docs/resources/Webhook.md

```diff
@@ -152,6 +152,8 @@ Add a new webhook to your GitHub repo (in the repo's settings), and use this end
 
 ## Edit Webhook Message % PATCH /webhooks/{webhook.id#DOCS_RESOURCES_WEBHOOK/webhook-object}/{webhook.token#DOCS_RESOURCES_WEBHOOK/webhook-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}
 
+Edits a previously-sent webhook message from the same token. Takes the same JSON parameters as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook).
```

Comment:
it doesn't take all the params execute webhook does; the params it does take are listed below

Reply:
Think this got messed up in the merge.